gulpfile.js

```diff
@@ -39,7 +39,13 @@ gulp.task('typings', function(done) {
 });
 
 gulp.task('tsc', function(done) {
-  runSpawn(done, 'node_modules/typescript/bin/tsc');
+  var tsc = spawn('node_modules/typescript/bin/tsc');
```

Comment:
Fixed. `spawn(task, opt_arg, {stdio: 'inherit'});` required that `opt_arg` to be defined as an array.